package.json

```
@@ -2,8 +2,9 @@
     "name": "counter",
     "version": "1.0.0",
     "main": "src/main.js",
+    "type": "module",
     "dependencies": {
-        "slint-ui": "^1.0.0"
+        "slint-ui": "^1.3.0-nightly.202311803"
```

For the release it can't be nightly:

```suggestion
        "slint-ui": "^1.3.0"
```

---

Therefore it is marked as todo on the description ;-)